dutch/check-mail/mail.md

```diff
@@ -1 +1,3 @@
-nope, je hebt geen mail
\ No newline at end of file
+Je leest nostalgisch een mail van je ex, nog een dringende mail van je baas, maar voor de rest, nope, geen mail...
+
+[Het bos](../bos/donker-bos.md) dan maar of [klimmen over het hek](../over-het-hek/hek.md)?
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.